test/TensorFlowRuntime/dataset_1.swift

```diff
@@ -63,15 +72,15 @@ public func model() {
     output_shapes: [TensorShape()]
   )
 
-  let one = getNextScalarFloatTensor(iterator).toHost(shape: [])
```

Comment:
the shape annotation in `.toHost(shape: [])` is not needed, because `getNextScalarFloatTensor()` now produces a CPU tensor, which can be sent to swift host without hoping through TPU, and thus need no shape annotation.